cwms-data-api/src/main/java/cwms/cda/api/TimeSeriesController.java

```diff
@@ -473,8 +514,49 @@ public void getOne(@NotNull Context ctx, @NotNull String id) {
                             + "of UTC shall be used.\r\nIgnored if version-date was specified with "
                             + "offset and timezone."),
                     @OpenApiParam(name = CREATE_AS_LRTS, type = Boolean.class, description = ""),
-                    @OpenApiParam(name = STORE_RULE,  description = "The business rule to use "
-                            + "when merging the incoming with existing data", type = StoreRule.class),
+                    @OpenApiParam(name = STORE_RULE,  type = StoreRule.class, description = "The business rule to use "
```

Comment:
For want of Java 17; but would it at least make sense to make this description string a constant so it's not duplicated?